tests/system/record_encoder/test_sentence_transformers_encoder.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+
+from canopy.knowledge_base.models import KBDocChunk
+from canopy.knowledge_base.record_encoder.sentence_transformers import (
+    SentenceTransformerRecordEncoder
+)
+from canopy.models.data_models import Query
+
+documents = [KBDocChunk(
+    id=f"doc_1_{i}",
+    text=f"Sample document {i}",
+    document_id=f"doc_{i}",
+    metadata={"test": i},
+    source="doc_1",
+)
+    for i in range(4)
+]
+
+queries = [Query(text="Sample query 1"),
+           Query(text="Sample query 2"),
+           Query(text="Sample query 3"),
+           Query(text="Sample query 4")]
+
+
+@pytest.fixture
+def encoder():
+    try:
+        encoder = SentenceTransformerRecordEncoder(batch_size=2)
+    except ImportError as e:
+        pytest.skip(
+            f"`transformers` extra not installed. Skipping SentenceTransformer system "
+            f"tests"
```

Comment:
```suggestion
            "`transformers` extra not installed. Skipping SentenceTransformer system "
            "tests"
```
The `f` prefix should not be used here